frontend/src/utils/logger.ts

```diff
@@ -0,0 +1,39 @@
+import { APPLICATION_ENV } from './credentials'
+
+/**
+ * This is the central logging configuration for the Nest.
+ * In development, logs are output to the console.
+ * In production, logs will be sent to Sentry (or a centralized logging service) for error tracking and monitoring.
+ * Sentry integration should be implemented here to ensure centralized and scalable error handling.
```

Comment:
@arkid15r 

Sentry integration should be written here so as to manage logs both in dev and prod env properly. Therefore added this comment.

Reply:
I believe Sentry has a documentation on how it should be set up. Not sure if that aligns w/ your expectations.

Reply:
Sentry integration is the scope of this https://github.com/OWASP/Nest/pull/176 .


let me know if have changes in scope this !